index.html

```diff
@@ -79,16 +80,10 @@
 
   <!-- Entrypoint -->
   <script>
-
-    // Create global app object
+    const MainMenu = require('./app/system/MainMenu')
     const App = require('./app/js/App')
-    const APP = new App()
-
-    // Create main menu (requires `APP` to be instantiated)
-    const {remote} = require('electron')
-    const {Menu} = remote
-    const menuTemplate = require('./app/system/menu')
-    Menu.setApplicationMenu(Menu.buildFromTemplate(menuTemplate))
+    const app = new App()
```

Comment:
You renamed `APP` to `app`, can you rename also the refs at `menuTemplate.js` in format and minify